src/pybkgmodel/data.py

```diff
@@ -312,11 +314,11 @@ def load_events(cls, file_name, cuts):
                         event_data[name] = data[key]
 
                     # Post processing
-                    event_data['true_zd'] = np.degrees(event_data['true_zd'])
-                    event_data['true_az'] = np.degrees(event_data['true_az'])
+                    event_data['true_zd'] = numpy.degrees(event_data['true_zd'])
```

Comment:
same as above

Reply:
As entire merge went wrong. I reset the merge. New version should not have this mistake.